### PR TITLE
Rebuild index.js after trailpacks installation

### PR DESCRIPTION
**Pull request by jaumard**
Fix https://github.com/trailsjs/generator-trails/issues/37

### CODE REVIEW

**Comment by jaumard** [comment]
Will fix https://github.com/trailsjs/trails/issues/149 and https://github.com/trailsjs/trails/issues/138

File: src/app/steps/writing.js [hunk 1]
```diff
@@ -57,16 +61,17 @@ export default {
...
       }
       trailpackNames.forEach(item => {
-        let ARCH = path.resolve(PROJECT_PATH + item, 'archetype', '**')
-        this.fs.copy(ARCH, dest)
+        let ARCH = path.resolve(PROJECT_PATH, item, 'archetype', '**')
+        fs.copy(ARCH, dest)
       })
+
+      //FIXME is there a better way for doing this ???
+      fs.commit(function(){
```
Comment (jaumard) [inline src/app/steps/writing.js]: This is a quick fix, I don't find any other way to do this... Normally generator don't have to (must not) call `commit`. But I don't find a way to scan `config` dir to rebuild index without commit first.

**Comment by jaumard** [comment]
Travis fail because of this bug https://github.com/trailsjs/trailpack-hapi/issues/76 when new trailpack-hapi will be release Travis will pass :)

**Comment by jaumard** [comment]
@trailsjs/maintainers can you look at this please ? Need to merge it asap and publish to npm to fix critical issues about new generate project. Thanks

**Comment by mikehostetler** [comment]
@jaumard We'll see what we can do to get this taken care of.  Thanks for bringing it to our attention.

**Comment by tjwebb** [comment]
Published in 1.0.0-beta-2

**Comment by jaumard** [comment]
Thanks @tjwebb :)

**Pull request merged**